template/sarif.json

```diff
@@ -9,13 +9,13 @@
           "semanticVersion": "3.2.2",
           "rules": [
             {
-              "id": "@cve-bin-tool/html_pdf_report",
-              "name": "HTML & PDF Scan reports by CVE Binary Tool",
+              "id": "@cve-bin-tool/html_pdf_sbom_report",
+              "name": "HTML, PDF & SBOM Scan reports by CVE Binary Tool",
               "shortDescription": {
-                "text": "HTML & PDF Scan reports by CVE Binary Tool"
+                "text": "HTML, PDF & SBOM Scan reports by CVE Binary Tool"
               },
               "fullDescription": {
-                "text": "HTML & PDF reports provide detailed insight of the CVEs and make them easy to track. These results get automatically updated eachtime when the CVE Binary Tool GitHub Action runs. You can disable this just by dismissing the alert, then you won't receive any further HTML/PDF reports."
+                "text": "HTML & PDF reports provide detailed insight of the CVEs and SBOM makes tracking dependency easy. These results get automatically updated eachtime when the CVE Binary Tool GitHub Action runs. You can disable this just by dismissing the alert, then you won't receive any further HTML/PDF/SBOM reports."
```

Comment:
```suggestion
                "text": "HTML & PDF reports provide detailed insight of the known vulnerabilities (CVEs) found.  The generated Software Bill of Materials (SBOM) file gives you a list of all the components found and scanned by cve-bin-tool.  This may not represent everything you know to to be in your software, only what we were able to identify.   These results get automatically updated each time the CVE Binary Tool GitHub Action runs. You can disable this by dismissing the alert, then you won't receive any further HTML/PDF/SBOM reports."
```

In a perfect world we'd follow up with a sentence encouraging people to file issues/make checkers for components that weren't found automatically , or explain how to add them to your SBOM.  I don't think we need that in this PR until we're actually past the beta stage, though.